v4l2-request-test.c

```diff
@@ -64,42 +64,51 @@ struct format_description formats[] = {
 
 static void print_help(void)
 {
-	printf("Usage: v4l2-request-test [OPTIONS] [SLICES PATH]\n\n"
+	printf("Usage: v4l2-request-test [OPTIONS]\n\n"
 	       "Options:\n"
-	       " -v [video path]                path for the video node\n"
-	       " -m [media path]                path for the media node\n"
-	       " -d [DRM path]                  path for the DRM node\n"
-	       " -D [DRM driver]                DRM driver to use\n"
-	       " -s [slices filename format]    format for filenames in the slices path\n"
-	       " -f [fps]                       number of frames to display per second\n"
-	       " -P [video preset]              video preset to use\n"
-	       " -i                             enable interactive mode\n"
-	       " -l                             loop preset frames\n"
-	       " -q                             enable quiet mode\n"
-	       " -h                             help\n\n"
-	       "Video presets:\n");
+	       " -v, --device <dev>\n"
```

Comment:
Could you try and keep the help text on the same line as the arguments?
Also, I don't really see the point of adding --device as a shorthand for --video-device but we can keep it if you think it's better that way.

Reply:
well, v4l2-compliance is using -d, --device for the video device. In v4l2-request-test -d goes with the drm device. That makes more sense to me. Having a correct destinction, --video-device is choosen. But admins/developers might be used to assume --device be the video device. Therfore both parameters are accecpted. I have no problem to remove --device, so.